dev/NavigationView/Strings/en-us/Resources.resw

```diff
@@ -153,8 +153,8 @@
     <value>More Button</value>
     <comment>Automation name for the nav view more button when panel is on top</comment>
   </data>
-  <data name="NavigationOverflowButtonText" xml:space="preserve">
-    <value>More</value>
-    <comment>Text for the nav view more button when panel is on top</comment>
+  <data name="NavigationOverflowButtonToolTip" xml:space="preserve">
+    <value>See more</value>
```

Comment:
Because we have #852 still in need of fixing, I recommend using "More" as the value of this tooltip.

And assuming that there's a UIA name for this button somewhere, that should also be "more"

Reply:
Changed the tooltip and UIA name for the overflow button to "More".